code/modules/medical/genetics/geneticsItems.dm

```diff
@@ -189,3 +175,78 @@
 		else
 			boutput(user, "<span class='alert'>You stab [M], but nothing happens.</span>")
 		return
+
+/obj/item/dna_scrambler
+	name = "dna scrambler"
+	desc = "An illegal retroviral genetic serum designed to randomize the user's identity, store it, and apply it later."
+	icon = 'icons/obj/syringe.dmi'
+	inhand_image_icon = 'icons/mob/inhand/hand_medical.dmi'
+	item_state = "syringe_0"
+	icon_state = "dna_scrambler_1"
+	force = 3
+	throwforce = 3
+	w_class = W_CLASS_SMALL
+	var/use_mode = "copy"
+	var/datum/bioHolder/bioHolder = new/datum/bioHolder
+	var/stored_name
+	contraband = 2
+
+	attack(mob/M as mob, mob/user as mob)
+		if(!M || !user)
+			return
+
+		if(src.use_mode == "depleted")
+			boutput(user, "<span class='alert'>The [name] is expended and has no more uses.</span>")
+			return
+
+		if(M == user)
+
+			if(use_mode == "copy")
+				src.copy_identity(user,user)
+				src.icon_state = "dna_scrambler_2"
+				user.visible_message("<span class='alert'><b>You inject yourself with the [src]! Your appearance has been copied to the [src].</b></span>")
+				src.use_mode = "paste"
```

Comment:
Since the lines following `copy_identity` are duplicated between the self-use and the other-person-use, I'd just stick all of this in `copy_identity`. Same for below.